src/app/detail/detail.component.ts

```diff
@@ -0,0 +1,43 @@
+import { HttpClient } from "@angular/common/http";
+import {Component, OnChanges, OnInit} from "@angular/core";
+import {ActivatedRoute} from "@angular/router";
+import {MorphicRecord} from "../interfaces";
+
+
+
+
+@Component({
+  selector: 'app-detail',
+  templateUrl: './detail.component.html',
+  styleUrls: ['./detail.component.scss']
+})
+export class DetailComponent implements OnInit  {
+
+
+
+  constructor(private http: HttpClient,private route: ActivatedRoute) {}
+
+  public rowData!: MorphicRecord;
```

Comment:
This looks like latest code, but there was the unused interface `MorphicRecord`. We should not use this one, if you could remove its uses and delete it, it would be great. `GridRecord` instead.

Reply:
I could not iterate GridRecord and take the subclass fields. I could not find any better solution this.